gumroad/views/admin.php

```diff
@@ -17,10 +17,63 @@
 }
 ?>
 
-<div class="wrap">
-
+<div class="wrap gumroad-wrap">
 	<h2><?php echo esc_html( get_admin_page_title() ); ?></h2>
 
+	<p>The Gumroad plugin makes it easy to use wordpress short codes to add links and embedded
+products from your Gumroad storefront directly into your wordpress pages and posts. There is
+a brand new post and page editor in Version 5.0 of wordpress and onward, Gumroad links and
+embed code will work the same, but only if they are added properly.</p>
+
+	<span style="margin-bottom:65px;display:block;"></span>
+	<h3 class="title"><?php esc_html_e( 'Important SSL Requirement', 'gum' ); ?></h3>
+
+	<p>
+		<b><em>
+			<?php esc_html_e( 'The address of the page hosting the Gumroad overlay or embed should start with https, ' .
+				'so you\'ll need to setup an SSL certificate for your site if you haven\'t already.', 'gum' ); ?>
+		</b></em>
+	</p>
+
+	<p>
+		If you need help setting up HTTPS on your Wordpress site here is <a href="http://wordpress.org/plugins/wordpress-https/" target="_blank">a Wordpress plugin</a> that can help.
+	</p>
+
+	<p>
+		<?php esc_html_e( 'Please refer to the Overlay and Embed documentation for additional help:', 'gum' ); ?>
```

Comment:
also remove all instances of esc_html_e – we don't use that anymore to keep thigns simple.